### ISSUE TITLE
Show undemangled c++ function names in diff result 

### ISSUE THREAD

**Issue by 4B5F5F4B** [joxeankoret/diaphora#135]
Diaphora shows undemangled c++ function names in diff result, for example:

![name1](https://user-images.githubusercontent.com/19218802/48673563-7ae3f800-eb7d-11e8-8989-929107776a61.PNG)

Undemangled c++ function names are not easy to understand, I think it would be much better if we can store demangled names instead of true names. So I modified a little code in function read_function defined in diaphora_ida.py like this:
```
def read_function(self, f, discard=False):
    name = GetFunctionName(int(f))
    true_name = name
    demangled_name = Demangle(name, INF_SHORT_DN)
    if demangled_name == "":
      demangled_name = None

    if demangled_name is not None:
      name = demangled_name
    #to store demangled names
    true_name = name
```

Now the diff result looks much better:

![name2](https://user-images.githubusercontent.com/19218802/48673627-4de41500-eb7e-11e8-962d-9df25e240ac6.PNG)

**Comment by joxeankoret** [comment]
I agree.

**Comment by joxeankoret** [comment]
Fixed with 6718a29bda7b1395d79f3eac19b2b1807285a1ce
